.travis.yml

```diff
@@ -17,7 +17,9 @@ before_install:
 install:
   - make install
 
+before_script:
```

Comment:
right now they are not much more than a linter step, which are normally run before tests. this way simple code style problems will fail the build immediately without running all the other (slow) tests. but happy to move this out of the PR.